backbone.js

```diff
@@ -979,14 +979,14 @@
     sort: function(options) {
       if (!this.comparator) throw new Error('Cannot sort a set without a comparator');
       options || (options = {});
+      var comparator = _.isFunction(this.comparator) ? _.bind(this.comparator, this) : this.comparator;
```

Comment:
Sure, but I think keeping this layer is useful for plugins/extensions built
on top of lodash that don't wish to break compatibility

On Mon, Aug 3, 2015 at 6:43 PM, John-David Dalton <notifications@github.com>
wrote:

> In backbone.js
> <https://github.com/jashkenas/backbone/pull/3732#discussion_r36140089>:
>
> > @@ -979,14 +979,14 @@
> >      sort: function(options) {
> >        if (!this.comparator) throw new Error('Cannot sort a set without a comparator');
> >        options || (options = {});
> > +      var comparator = _.isFunction(this.comparator) ? _.bind(this.comparator, this) : this.comparator;
>
> Because iteratee methods no longer have a context param punting for devs
> to just pass in iteratees pre-bound with _.bind or Function#bind.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jashkenas/backbone/pull/3732/files#r36140089>.
>